docs/support/known-issues.md

```diff
@@ -0,0 +1,40 @@
+# Known Issues
+
+This page informs about known issues or bugs in the different artifacts and templates we provide.
+
+::: info Information
+If you found an issue not mentioned here, please head over to our GitHub repositories and check if a corresponding bug ticket exists.
+If not feel free to open a new one.
+:::
+
+## API gateway
+
+- No known issues.
```

Comment:
I would not list that instead of writing "No known issues." as that isn't often correct but there are bugs in GitHub. Even if you mention it above i find that misleading.